shared/src/main/scala/scala/util/parsing/combinator/Parsers.scala

```diff
@@ -792,6 +792,57 @@ trait Parsers {
       applyp(in)
     }
 
+  /** A parser generator for a specified range of repetitions interleaved by a
+   * separator.
+   *
+   *  `repN(n, m, p, s)` uses `p` at least `m` times and up to `n` times, interleaved
```

Comment:
```suggestion
   *  `repN(m, n, p, s)` uses `p` at least `m` times and up to `n` times, interleaved
```

Reply:
fixed.

Reply:
following @SethTisue suggestion, I renamed the function to `repNM`.